tests/common.c

```diff
@@ -143,20 +142,20 @@ write_vectored_cb(void *userdata, const rustls_iovec *iov, size_t count,
 }
 #endif /* _WIN32 */
 
-size_t
-bytevec_available(bytevec *vec)
+inline size_t
+bytevec_available(const bytevec *vec)
 {
   return vec->capacity - vec->len;
 }
 
-char *
-bytevec_writeable(bytevec *vec)
+inline char *
+bytevec_writeable(const bytevec *vec)
 {
   return vec->data + vec->len;
 }
 
-void
-bytevec_consume(bytevec *vec, size_t n)
+inline void
```

Comment:
It fell away in the last commit (sorry, not my tidiest git history) because Fun Fact :tm: : while gcc and clang will let you `inline` across compilation unit boundaries at any optimization level it seems like msvc will only let this pass without optimization. The Windows release builds were all failing with missing symbol errors until I undid this :dizzy_face: 

In the end I think you're right and it was a pointless exercise :laughing:

Reply:
Removed `inline` and also undid the part in the last commit where I made these `static` and implemented in the `.h`. The pragma hoop jumping needed to silence the resulting unused warning from `server.c` makes it not worthwhile in retrospect.